app/components/directives/settings/ldapsetting.html

```diff
@@ -126,10 +127,25 @@
                                     </label>
                                 </div>
                             </div>
-
+                        </div>
+                        <div class="ui row" *ngIf="!ldapConfig.insecure_skip_verify && ldapConfig.start_tls">
+                            <div class="ui six wide column">
+                                <div class="field" [ngClass]="{error: tls_cert_issued_to.errors?.required && formRef.submitted}">
+                                    <label for="service_account_dn">TLS Cert issued to</label>
```

Comment:
What is TLS Cert verifier? It is not clear what to enter as value of this field from the label.

Also typo in `<label for="...">` Field id is incorrect.